cpp/src/io/parquet/writer_impl.cu

```diff
@@ -1572,7 +1650,72 @@ void writer::impl::write(table_view const& table, std::vector<partition_info> co
     }
   }
 
-  // Build chunk dictionaries and count pages
+  // until now we've been using a uniform fragment size. if there are any columns with
```

Comment:
I would be a fan of rewording this. In 2 years this comment will be odd to read. Maybe Before 23.04..."

Reply:
will do...was actually meaning "up until this point in the function", but I can see the source of confusion